foodiefunreact/src/components/AddNewRestaurant.js

```diff
@@ -21,7 +21,7 @@ const AddNewRestaurant = props => {
     event.preventDefault();
 
     axiosWithAuth()
-    .post('newRestaurant', restaurantInfo)
+    .post(`user/restaurants`, restaurantInfo)
```

Comment:
Not sure if you meant to leave it as back-ticks or quotations

Reply:
I believe it will need to be backticks because we will be using something like ${user_id} to post the restaurant info to the correct user

Reply:
Ok, yea that makes sense